src/practice_menu.cpp

```diff
@@ -1,487 +1,46 @@
-                default_load();
-                camera = {1313.54285f, -234.203003f, 5545.16846f, 1027.53259f, -108.096123f, 5605.23047f, 0.0f, 318.295868f};
-                angle = 16384;
-                position = {1309.60645f, -240.0f, 5533.43848f};
-                practice_file.inject_options_after_load = set_camera_angle_position;
-                practice_file.inject_options_after_counter = 10;
-                break;
-            }
-            case AERALFOS_SKIP_INDEX: {
-                loadFile("tpgz/save_files/aeralfos_skip.bin");
-                default_load();
-                break;
-            }
-            case CITS_2_INDEX: {
-                loadFile("tpgz/save_files/cits_2.bin");
-                default_load();
-                break;
-            }
-            case FAN_TOWER_INDEX: {
-                loadFile("tpgz/save_files/fan_tower.bin");
-                default_load();
-                break;
-            }
-            case ARGOROK_INDEX: {
-                loadFile("tpgz/save_files/argorok.bin");
-                default_load();
-                practice_file.inject_options_during_load = argorok;
-                break;
-            }
-            case PALACE_1_INDEX: {
-                loadFile("tpgz/save_files/palace_1.bin");
-                default_load();
-                break;
-            }
-            case PALACE_2_INDEX: {
-                loadFile("tpgz/save_files/palace_2.bin");
-                default_load();
-                practice_file.inject_options_after_load = palace2;
-                break;
-            }
-            case EARLY_PLATFORM_INDEX: {
-                loadFile("tpgz/save_files/early_platform.bin");
-                default_load();
-                break;
-            }
-            case ZANT_INDEX: {
-                loadFile("tpgz/save_files/zant.bin");
-                default_load();
-                break;
-            }
-            case HC_INDEX: {
-                loadFile("tpgz/save_files/hc.bin");
-                default_load();
-                break;
-            }
-            case DARKNUT_INDEX: {
-                loadFile("tpgz/save_files/darknut.bin");
-                default_load();
-                break;
-            }
-            case HC_TOWER_INDEX: {
-                loadFile("tpgz/save_files/hc_tower.bin");
-                default_load();
-                break;
-            }
-            case BEAST_GANON_INDEX: {
-                loadFile("tpgz/save_files/beast_ganon.bin");
-                default_load();
-                break;
-            }
-            case HORSEBACK_GANON_INDEX: {
-                loadFile("tpgz/save_files/horseback_ganon.bin");
-                default_load();
-                break;
-            }
-        }
-    }
-
-    Utilities::move_cursor(cursor, LINES);
-    Utilities::render_lines(font, lines, cursor.x, LINES);
+	if (!init_once) {current_input = 0;init_once = true;}
```

Comment:
There is a clang format for the project, though I don't know if anyone else is using it. This line probably should be written across multiple lines if you don't want to run the formatter.